app/controllers/people/picture_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module People
+  class PictureController < CrudController
+
+    self.permitted_attrs = %i[picture]
+
+    def update
+      person.update!(picture: params[:picture])
+      render json: { data: { picture_path: picture_person_path(params[:id]) } }
+    end
+
+    def show
+      picture_url = person.picture.file.nil? ? default_avatar_url : person.picture.url
```

Comment:
I don't like the idea that the whole file path is stored in person.picture.url. As I mentioned, a hacker could use this to download any file from the running machine/container if he's able to change that path. 
maybe do something similar like in the default_avatar_url ....